src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin dimensions {
```

Comment:
The name of the mixin 'dimensions' is too generic. It would be better to give it a more specific name relating to its purpose, for example 'starDimensions'.